components/docs-chef-io/content/automate/High_Availability.md

```diff
@@ -0,0 +1,301 @@
++----------------------+-----------------------------------------------+
+| Ubuntu (64 Bit OS)   |  14.04.x, 16.04.x, 18.04.x, 20.04.x           |
++----------------------+-----------------------------------------------+
+| Centos (64 Bit OS)   |  7                                            |
++----------------------+-----------------------------------------------+
+
+## Virtual Machine (VM) instances Type 
+
++--------------+-------------+--------------------------+--------------+
+|              |       Type  |                    RAM   |              |
+| Instance     |             |                          | Volume-size  |
++==============+=============+==========================+==============+
+| PostgreSQL   | t3a.medium  | 4GB RAM For Test and 8G  | 50GB         |
+|              |             | For Prod                 | (dedicated   |
+|              |             |                          | hard disk    |
+|              |             |                          | space need   |
+|              |             |                          | to be        |
+|              |             |                          | assigned to  |
+|              |             |                          | '/')         |
++--------------+-------------+--------------------------+--------------+
+| El           | m5a.large   | 8GB RAM For Test and 16G | 50G          |
+| asticsearch  |             | For Prod                 | B (dedicated |
+|              |             |                          | hard disk    |
+|              |             |                          | space need   |
+|              |             |                          | to be        |
+|              |             |                          | assigned to  |
+|              |             |                          | '/')         |
+|              |             |                          |              |
+|              |             |                          |              |
++--------------+-------------+--------------------------+--------------+
+| Automate     | t3a.medium  | 4GB RAM For Test and 8G  | 50G          |
+|              |             | For Prod                 | B (dedicated |
+|              |             |                          | hard disk    |
+|              |             |                          | space need   |
+|              |             |                          | to be        |
+|              |             |                          | assigned to  |
+|              |             |                          | '/')         |
+|              |             |                          |              |
+|              |             |                          |              |
++--------------+-------------+--------------------------+--------------+
+| Chef Server  | t3a.medium  | 4GB RAM For Test and 8G  | 50G          |
+|              |             | For Prod                 | B (dedicated |
```

Comment:
it should also be t3.medium instead of t3a.medium

Reply:
done